common/src/main/java/com/microsoft/identity/common/internal/eststelemetry/LastRequestTelemetry.java

```diff
@@ -47,7 +50,7 @@
     }
 
     List<FailedRequest> getFailedRequests() {
-        return failedRequests;
+        return Collections.unmodifiableList(failedRequests);
```

Comment:
This doesn't take a safe-copy, right?

Reply:
Not sure what you mean here, can you tell more?

Reply:
This is more about being sure that whoever is using this is OK with the list changing underneath them.

Reply:
Yes, they are okay with that. Added javadoc here as well about being unmodifiable: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1018/commits/d71f93b10e05f31e6d3768f07117004b30837621